src/lib/mina/mina-instance.ts

```diff
@@ -20,6 +35,11 @@ export {
   activeInstance,
   setActiveInstance,
   ZkappStateLength,
+  reportGetAccountError,
```

Comment:
Moved these methods here as they are shared between `local-blockchain` and `remote`